types/animation.d.ts

```diff
@@ -0,0 +1,37 @@
+import { PlayerObject } from './model'
+
+declare function invokeAnimation(
+  animation: CompositeAnimation,
```

Comment:
The type of `animation` can be `CompositeAnimation` or `function(player: PlayerObject, time: number) : void`.

Reply:
Maybe we can define a type named `Animation` which includes the two cases above?

Reply:
This is function overload. No need to add a type alias.